pygmt/datasets/earth_relief.py

```diff
@@ -101,17 +103,20 @@ def load_earth_relief(
     Examples
     --------
 
+    >>> from pygmt.datasets import load_earth_relief
     >>> # load the default grid (gridline-registered 01d grid)
     >>> grid = load_earth_relief()
```

Comment:
I feel for the sake of clarity it's good to state the unit in long form.

```suggestion
    >>> # load the default grid (gridline-registered 1 arc-degree grid)
    >>> grid = load_earth_relief()
```